Assets/Scripts/OncominObjects/LevelPart.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class LevelPart : MonoBehaviour
+{
+    [SerializeField] private float _speed;
+    [SerializeField] private float _destroyTime;
+
+    private Player _player;
+
+    private bool _isArrival;
+
+    public event UnityAction OnArrival;
```

Comment:
https://trello.com/c/CsOZfMhi/1-%D0%BD%D0%B0%D0%B7%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2